acorn.js

```diff
@@ -2393,6 +2393,7 @@
     classBody.body = [];
     expect(_braceL);
     while (!eat(_braceR)) {
+      if (eat(_semi)) continue;
```

Comment:
Ah, you mean performance. Your suggestion introduced a bug but yes, it is slightly more efficient in this rather insignificant case. I do not feel that is an improvement.